lib/data-client.js

```diff
@@ -43,7 +43,34 @@ class SourcebitDataClient {
 
         await cacheFileExists;
 
-        return fse.readJson(DEFAULT_FILE_CACHE_PATH);
+        return new Promise((resolve, reject) => {
+            // Cache file might be in the middle of being written, hence readJson might fail, so retry after a short delay.
+            const retryDelay = 5;
```

Comment:
* 1000

Reply:
@davbree it is for fixing race conditions when it tries to accses a file while it is still being written transform function of sourcebit-target-next. I think the file writing operation shouldn't take more than couple milliseconds for really large JSONfiles.
@rodikh can you check how long does it take to write huge file, like 100MB. Then multiply by some factor (maybe 10) and use it as the delay time.

Reply:
I think, at the end of the day, this will be dictated by the hardware disc speed. Testing on my SSD mac would yield different results to whatever a dockerized and virtualized server might have.